src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -223,10 +224,17 @@ layout program_node::calc_output_layout() const {
     if (allow_new_shape_infer) {
         auto out_layouts = type()->calc_output_layouts(*this, *get_kernel_impl_params());
         if (!out_layouts.empty()) {
+            auto out_layout = out_layouts[0];
+            if (is_type<crop>() && (out_layouts.size() > 1)) {
```

Comment:
Please add some comments here 

"Since multiple output is not supported yet, currently only crops created by Split and VariadicSplit are implemented as a workaround to return all original output layouts. So this part should be refactored once multiple output is supported and the splits are implemented for mulitple output. "

Reply:
Done